scripts/scil_visualize_bundles_mosaic.py

```diff
@@ -0,0 +1,248 @@
+
+from scilpy.io.utils import (
+    add_overwrite_arg, assert_inputs_exist, assert_outputs_exists)
+
+from dipy.viz import actor, window
+from PIL import Image
+from PIL import ImageFont
+from PIL import ImageDraw
+
+
+def build_arg_parser():
+    parser = argparse.ArgumentParser(
+        description='Visualize bundles from a list. '
+                    'The script will output one screenshot per direction '
+                    '(6 total) for each bundle in the list.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('anat_reference',
+                        help='Image used as background (e.g. T1, FA, b0).')
+    parser.add_argument('inputs', nargs='+',
+                        help='List of streamline files supported by nibabel.')
+    parser.add_argument('output_name',
+                        help='Name of the output image mosaic (e.g. jpg, png).')
+    parser.add_argument('--zoom', type=float, default=1.0,
+                        help='Rendering zoom. '
+                        'A value greater than 1 is a zoom-in, a value less than 1 is a zoom-out.')
+    parser.add_argument('--ttf', default=None,
+                        help='Path of the true type font to use for the legends.')
+    parser.add_argument('--ttf_size', type=int, default=35,
+                        help='Font size (int) to use for the legends.')
+    parser.add_argument('--opacity_background', type=float, default=0.4,
+                        help='Opacity of background image [0, 1.0]')
+
+    add_overwrite_arg(parser)
+    return parser
+
+
+def main():
+    parser = build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.anat_reference])
+    assert_outputs_exists(parser, args, [args.output_name])
+
+    output_names = ['axial_superior', 'axial_inferior',
+                    'coronal_posterior', 'coronal_anterior',
+                    'sagittal_left', 'sagittal_right']
+
+    listOfBundles = [f for f in args.inputs]
+    output_dir = os.path.dirname(args.output_name)  # Where temporal files will be created
+    # ----------------------------------------------------------------------- #
+    # Mosaic, column 0: orientation names and data description
+    # ----------------------------------------------------------------------- #
+    width = 300
+    height = 300
+    rows = 6
+    cols = len(listOfBundles)
+    text_pos_x = 50
+    text_pos_y = 50
+
+    # Creates a new empty image, RGB mode
+    mosaic = Image.new('RGB', ((cols + 1) * width, (rows + 1) * height))
+    # Prepare draw and font objects to render text
+    draw = ImageDraw.Draw(mosaic)
+    if args.ttf is not None:
+        try:
+            font = ImageFont.truetype(args.ttf, args.ttf_size)
+        except:
+            print('Font %s was not found. Default font will be used.' % (args.ttf))
+            font = ImageFont.load_default()
+    else:
+        font = ImageFont.load_default()
+
+    # T1 as background
```

Comment:
Might not be a T1.

Reply:
"Data of the image used as background" ??

Reply:
Makes more sense :)